backend/migrations/20230724125220_trigram.up.sql

```diff
@@ -0,0 +1,4 @@
+-- Add up migration script here
+
+CREATE EXTENSION pg_trgm;
+SET pg_trgm.similarity_threshold = 0.8;
```

Comment:
Remove as there's no longer a need for? If we don't get it to run, we should remove it.